doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,742 @@
+    ")\n",
+    "\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='depth',\n",
+    "    cmap='plasma_r',\n",
+    "    cnorm='log',\n",
+    "    frame_width=width,\n",
+    "    title=\"Earthquakes with log color scaling\"\n",
+    ")\n",
+    "\n",
+    "plot3 = df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='depth',\n",
+    "    cmap='plasma_r',\n",
+    "    cnorm='eq_hist',\n",
+    "    frame_width=width,\n",
+    "    title=\"Earthquakes with Histogram Equalization\"\n",
+    ")\n",
+    "\n",
+    "(plot1 + plot2 + plot3).cols(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "881751c3-4dc2-4027-9b7f-147fbe64a4f6",
+   "metadata": {},
+   "source": [
+    "(option-fontscale)=\n",
+    "## `fontscale`\n",
+    "\n",
+    "The `fontscale` option scales all the fonts in the plot by the provided numeric factor. For example, setting `fontscale=1.5` enlarges the title, tick labels, and axis labels by 50%. This is useful when you want to emphasize text for presentations or detailed viewing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1ddd044-1771-43a3-a09d-580749371644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    fontscale=1.5,\n",
+    "    title=\"Penguins Species\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee479a25-5270-4b60-91c3-5da24ae2a501",
+   "metadata": {},
+   "source": [
+    "(option-fontsize)=\n",
+    "## `fontsize`\n",
+    "\n",
+    "The `fontsize` option sets the font size for different text elements in the plot. It can be:\n",
+    "\n",
+    "- A single value (e.g. `12`) to apply to all text elements.\n",
+    "- A dictionary to control specific elements like the title, axis labels, and ticks.  \n",
+    "  Example: `{'title': '15pt', 'xlabel': '12pt', 'ylabel': '12pt', 'ticks': 10}`\n",
+    "\n",
+    "This option is useful for precise text control in reports, presentations, and dashboards.\n",
+    "\n",
+    "::: {note}\n",
+    "**Backend-specific behavior**\n",
+    "\n",
+    "- The **Bokeh** backend supports both numeric values and strings with units like `'12pt'` or `'10px'`.\n",
+    "- The **Matplotlib** backend only accepts **numeric values**. Using strings like `'14pt'` will not work.\n",
+    "\n",
+    "To ensure compatibility across backends, prefer using numbers unless you're specifically targeting Bokeh.\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "670232e8-34be-43bd-a07b-0604b3b26eb6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
```

Comment:
Not sure this cell is required here?

Reply:
Which cell?

Reply:
That's the cell where the penguins dataset is imported for the examples. If I remove it there, then I'll have to import it twice (once each for the bokeh and mpl examples).

Reply:
Ok I understand. What confused me was the `.head(3)` call, no sure why it's there in the middle of this reference page?

<img width="724" alt="image" src="https://github.com/user-attachments/assets/37c5bbb9-5572-45c3-b44d-741fd886f508" />

Reply:
OK. Since `df` there serves both plots which are not in the same cell, I wanted to show a snippet of the dataframe before its plotted.

Reply:
I still think it's strange to call `.head(3)` in the middle of this page.

Reply:
OK. I can remove that line then.